src/components/ProTips/index.tsx

```diff
@@ -0,0 +1,136 @@
+                            </div>
+                        </div>
+                    </div>
+                    <div className="grid grid-cols-1 md:grid-cols-2 gap-12 mb-8">
+                        <div className="flex">
+                            <div className="outline outline-gray-200 rounded-full w-16 h-16 me-3">
+                                <div className="m-5">
+                                    <LiaPencilRulerSolid size='1.5rem' color={`${isDarkMode ? 'white' : 'black'}`} />
+                                </div>
+                            </div>
+                            <div className="">
+                                <p className='font-semibold dark:text-white'>Exclusive downloads</p>
+                                <div className="flex mt-2">
+                                    <span className=''></span>
+                                    <div className="flex outline outline-gray-200 outline-1 py-1 rounded">
+                                        <img className="object-cover w-5 h-5 rounded mr-1" src="/logo192.png" alt="" />
+                                        <p className='text-sm font-bold text-gray-400'>2 mins read</p>
+                                    </div>
+                                </div>
+                            </div>
+                        </div>
+                        <div className="flex">
+                            <div className="outline outline-gray-200 rounded-full w-16 h-16 me-3">
+                                <div className="m-5">
+                                    <BsCameraVideo size='1.5rem' color={`${isDarkMode ? 'white' : 'black'}`} />
+                                </div>
+                            </div>
+                            <div className="">
+                                <p className='font-semibold dark:text-white'>Behind the scenes</p>
+                                <div className="flex mt-2">
+                                    <span className='text-sm font-bold rounded bg-orange-300/50 py-1 px-2 mr-1'>Hot</span>
+                                    <div className="flex outline outline-gray-200 outline-1 py-1 rounded">
+                                        <img className="object-cover w-5 h-5 rounded mr-1" src="/logo192.png" alt="" />
+                                        <p className='text-sm font-bold text-gray-400'>3 mins read</p>
+                                    </div>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                    <div className="grid grid-cols-1 md:grid-cols-2 gap-12 mb-8">
+                        <div className="flex">
+                            <div className="outline outline-gray-200 rounded-full w-16 h-16 me-3">
+                                <div className="m-5">
+                                    <BsTelephone size='1.5rem' color={`${isDarkMode ? 'white' : 'black'}`} />
```

Comment:
Hey @UlrichFlynn, what you've done here is correct. But you could change icon font-color in darkmode/ lightmode in it's className attribute like below

`<BsTelephone size='1.5rem' className="text-gray-800 dark:text-white" />`

Reply:
Ok